communication/moderators.md

```diff
@@ -12,31 +12,35 @@ communication channels and their home time zone. See our
 
 Administrator seats: 3
 
-- Paris Pittman  - PT
-- Aaron Crickenberger - PT
-- Open
+| Name                | Kubernetes Slack ID | Timezone                                                | UTC offset |
+|---------------------|---------------------|---------------------------------------------------------|------------|
+| Paris Pittman       | @paris              | [PT - Pacific Time (US West Coast)](https://time.is/PT) | − 08:00    |
+| Aaron Crickenberger | @spiffxp            | [PT - Pacific Time (US West Coast)](https://time.is/PT) | − 08:00    |
+| *Open*              | *Open*              |                                                         |            |
 
 #### Primary Moderators
 
 Primary moderators seats: 7
 
-- Jorge Castro (@castrojo) - ET
-- Jaice Singer DuMars - (@jdumars) - PT
-- Louis Taylor (@kragniz)- CET
-- Ihor Dvoretskyi (@idvoretskyi) - EET
-- Nikhita Raghunath (@nikhita) - IST
-- Bob Killen (@mrbobbytabkes) - ET
-- Open
+| Name                | Kubernetes Slack ID | Timezone                                                | UTC offset |
+|---------------------|---------------------|---------------------------------------------------------|------------|
+| Jorge Castro        | @castrojo           | [ET - Pacific Time (US East Coast)](https://time.is/ET) | − 05:00    |
+| Bob Killen          | @mrbobbytables      | [ET - Pacific Time (US East Coast)](https://time.is/ET) | − 05:00    |
+| Jaice Singer DuMars | @jdumars            | [PT - Pacific Time (US East Coast)](https://time.is/PT) | − 08:00    |
+| Louis Taylor        | @kragniz            | [CET - Central European Time](https://time.is/CET)i     | + 01:00    |
+| Ihor Dvoretskyi     | @ihor.dvoretskyi    | [EET - Easter European Time](https://time.is/EET)       | + 02:00    |
+| Nikhita Raghunath   | @nikhita            | [IST - India Standard Time](https://time.is/India)      | + 05:30    |
```

Comment:
s/India Standard Time/Indian Standard Time 

Though you may find the former online too, the latter is more commonly used.